docs/experiment/experiment_introduction.md

```diff
@@ -0,0 +1,23 @@
+# Experiment Introduction
+
+After a successful login you will be redirected to your personal landing page.
+From there, [navigate](../project/project_introduction.md#project-navigation) into your project of interest to see the project summary.
+In the project summary view you can either [navigate](#experiment-navigation) into an experiment
```

Comment:
this links two sentences down, not sure if necessary

Reply:
I suspect that the introduction will change anyway and would keep the navigation aspect if people are truly lost for now 🤔